### ISSUE TITLE
Constant queries to __lastCheckTime.txt

### ISSUE THREAD

**Issue by Dorus** [docker-library/drupal#195]
I've set up the docker image on microsoft azure. I'm using an app service to host the image, a storage account to mount 

/var/www/html/modules
/var/www/html/sites
/var/www/html/profiles
/var/www/html/themes

Also i'm using an Azure Database for MySQL flexible server as database. This one was hard to set up, but i managed to do it.

Since (i think), the 4th this month, i suddenly had a huge spike in usage of the storage account. Cost of the storage account jumped up to almost half my total running cost.

Checking the logs, there is a constant ongoing attempt to query gsite\\__lastCheckTime.txt, profiles\\__lastCheckTime.txt and themes\\__lastCheckTime.txt (but not modules). It's just a constant repeating of queryInfo, Create and Close calls (aka QueryAllInformation, FileOpen and FileClose), around 200 per minute. Notice the modules dir is missing.

This has been ongoing for almost 2 weeks now. And i dont remember doing anything special with the site when it started. Possibly i did reboot and update the image? Subsequent reboots haven't solved it.

**Comment by wglambert** [comment]
Seems like something specific to Azure
![image](https://user-images.githubusercontent.com/26428991/122473288-7f136480-cf76-11eb-9861-e0aadc7fe8e7.png)

Can't find anything drupal related with that filename
![image](https://user-images.githubusercontent.com/26428991/122473296-8175be80-cf76-11eb-9fc5-473c85537f91.png)

And there's nothing in the container with that name
```console
$ docker run -it --rm drupal bash
root@40340d7855b7:/opt/drupal# find / | grep -i lastchecktime
root@40340d7855b7:/opt/drupal#
```

But I can't find anything more about it other than unrelated help issues

**Comment by Dorus** [comment]
Well thanks for looking anyway. I couldn't find anything at all myself so far.

Only place i've ever seen __lastCheckTime.txt before is in the /LogFiles folder on the app storage sftp

![afbeelding](https://user-images.githubusercontent.com/4461459/122477836-a8f57700-cfc8-11eb-95b6-46e2407e13ac.png)

I've already tried various things like disabling the file logs, adding the __lastCheckTime.txt to the storage account.

Also i've now removed the empty profile and theme fileshare to save cost. I just noticed the modules is there, just less often.

Possible i need to raise an report with azure.
